core/bin/zksync_api/src/fee_ticker/validator/mod.rs

```diff
@@ -233,12 +233,12 @@ mod tests {
     #[ignore]
     // We can use this test only online, run it manually if you need to test connection to uniswap
     async fn get_real_token_amount() {
-        let mut watcher = UniswapTokenWatcher::new(
-            "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2".to_string(),
-        );
-        let dai_token_address =
-            Address::from_str("6b175474e89094c44da98b954eedeac495271d0f").unwrap();
-        let dai_token = Token::new(TokenId(1), dai_token_address, "DAI", 18, TokenKind::ERC20);
+        const MAINNET_RIF_CONTRACT_ADDR: &str = "0x2acc95758f8b5f583470ba265eb685a8f45fc9d5";
```

Comment:
did check if the token_address have the same format, since dai use it without `0x` and rif has it.

Reply:
good observation. The `Address::from_str` on line 240 will parse it into H160 whether it is prefixed with 0x or not

Reply:
I wouldn't store the contract address in the code, while I'd specify it with a config param

Reply:
this is just for the unit test and this const is only available within the test block. What would be the reasons to not use it here directly?

Reply:
changed (actually had to remove the 0x prefix after all @franciscotobar :D)